k8_cortx_cloud/solution.yaml

```diff
@@ -3,11 +3,10 @@ solution:
   secrets:
     name: cortx-secret
     content:
-      openldap_admin_secret: seagate1
       kafka_admin_secret: Seagate@123
       consul_admin_secret: Seagate@123
       common_admin_secret: Seagate@123
-      s3_auth_admin_secret: ldapadmin
+      s3_auth_admin_secret: cortxadmin
```

Comment:
I don't know why the default password is `ldapadmin`.  I am changing it to the more reasonable `cortxadmin`.